cmd/context/create.go

```diff
@@ -282,6 +283,33 @@ func (c *Command) initOktetoContext(ctx context.Context, ctxOptions *Options) er
 	okteto.GetContext().IsTrial = clusterMetadata.IsTrialLicense
 	okteto.GetContext().CompanyName = clusterMetadata.CompanyName
 
+	if len(clusterMetadata.CliVersionRange) > 0 {
+		skipCheck, ok := os.LookupEnv("OKTETO_SKIP_CLUSTER_CLI_VERSION")
```

Comment:
Could we use `env.LoadBoolean()`? It covers all the possible true values

Reply:
+1 also for consistency with other feature toggles we use booleans instead of integers